ads-with-uniregistry.rst

```diff
@@ -0,0 +1,73 @@
+using the code ``DOCS25``.
+We hope that you'll enjoy getting started with the service,
+and find it as useful.
+`Sign up`_ now to get started.
+
+Here are some of the reasons that Uniregistry is a good domain host:
+
+Usability
+---------
+
+Uniregistry makes every interaction point simple, minimal and as sharp looking as possible. This pays off in a good complete experience that should delight you. It doesn’t suck.
+
+Domain registration starts with search, and their clutter-free, fast-loading search gets the job done across hundreds of TLDs worldwide. They make everything you might need to do possible within one or two clicks, instead of 10.
+
+They also have native apps on iOS and Android that work well, and make it easy to grab a new domain, manage your existing names, or transfer in and out at your leisure.
+
+Privacy and Security
+--------------------
+
+Uniregistry offers free WHOIS Privacy on every domain. No nickle-and-diming to protect your personal information. In recent years, those of you who have your phone numbers and emails on domain registration have come to experience being heckled by automated phone calls and spam. Uniregistry takes this off the table with no extra cost to you.  Uniregistry’s competitors charge between $3.99 and $7.99 for what should be your right, not a paid privilege.
+
+Two step verification is built right into the product as well. They are the registrar for people who own one domain or thousands, so they had to build in security and tools to put you in complete control. Since they have handled the use case of the most demanding domain owners in the industry, you get the same benefits even if you just buy one.
+
+Price
+-----
+
+Our analysis showed that most domain registrars which provide good pricing are constantly up-selling and playing renewal games that cost money.  Uniregistry offer transparent, honest pricing on your most valuable assets. They have built a business like we would, they have never charged its customers for WHOIS privacy, they believe that freedom from unwelcome solicitation or online harassment is a fundamental right, not an opportunity for an upsell.
+
+Uniregistry is hyper-focused on domain registration. They don’t offer hosting, so you are not being up-sold constantly. Their pricing across the board is competitive, while providing a first class experience. 
+
+Additionally, we worked with Uniregistry to give you a discount for not only new domain purchases, but also your transfers. Use this link as our thank you for supporting open source through your domain registration. `Try it`_ and see if you agree.
+
+Transparency
+------------
+
+Uniregistry doesn't hide things like auth codes from you. Click and you shall receive. Domain renewals are a major source of cheap profit for other hosts, but it's quite simple to switch with Uniregistry. They provide the transparency required to be a long term partner to you in managing your online presence.
```

Comment:
"auth codes" could use some clarification, I wasn't sure of what you were describing until the end of the paragraph here. Maybe make this something about making transfers more difficult.

"Click and you shall receive" is cute. I'm perfectly on the fence here, but I'm leaning towards this being a bit too far towards marketing cute :)